### PR TITLE
93 - add language step

### LINKED ISSUE #93: (SP:2) Implement language selection
**User Story: #11** 

## Description

Implement language selection at `Language Step` at stepper.

## Design

Image

Image

### PR DESCRIPTION
**Pull request by natamrshn**
image

image

add language step at stepper

### CODE REVIEW

**Review by lublubczyk — APPROVED** [review]

**Review by nadraharr — APPROVED** [review]

**Review by VoitV — APPROVED** [review]

File: src/containers/tutor-home-page/language-step/LanguageStep.jsx [hunk 1]
```diff
@@ -1,17 +1,48 @@
...
     <Box sx={styles.container}>
       <Box sx={styles.imgContainer}>
         <Box component='img' src={img} sx={styles.img} />
       </Box>
       <Box sx={styles.rigthBox}>
-        <Box>Language here</Box>
-        {btnsBox}
+        <Box sx={styles.contentBox}>
+          <Box sx={styles.textBox}>
+            Please select the language in which you would like to study and
```
Comment (AntonRomanov999) [inline src/containers/tutor-home-page/language-step/LanguageStep.jsx]: May be use translation here? and update it if needed...

Comment (vlad-khrychov) [reply src/containers/tutor-home-page/language-step/LanguageStep.jsx]: I agree with comment, use translation for all labels and titles.

File: src/containers/tutor-home-page/language-step/LanguageStep.jsx [hunk 2]
```diff
@@ -1,17 +1,48 @@
...
+          <AppSelect
+            MenuProps={{
+              PaperProps: {
+                style: {
+                  maxHeight: 300
+                }
+              }
+            }}
+            fields={languageFields}
+            label='Your native language'
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/language-step/LanguageStep.jsx]: Also, maybe add a translation here?

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: src/containers/tutor-home-page/language-step/LanguageStep.jsx (hunk 1 above)
Comment (natamrshn) [reply src/containers/tutor-home-page/language-step/LanguageStep.jsx]: Fix this bug)

**Review by AntonRomanov999 — APPROVED** [review]

**Pull request merged**